src/modules/user/apiKeys/reducer.ts

```diff
@@ -42,13 +44,15 @@ export const apiKeysReducer = (state = initialApiKeysState, action: ApiKeysActio
                 ...state,
                 apiKeys: sliceArray(action.payload.apiKeys, defaultStorageLimit()),
                 dataLoaded: true,
+                error: undefined,
```

Comment:
add please also dataLoaded: true, for each successful action

Reply:
dataLoaded for fetching,
added success value for create, delete, update